test/fixtures/recordings/Integration-Test_519117731/Test-publish-function-locally_1268185746/recording.har

```diff
@@ -4930,6 +4930,105 @@
           "wait": 825
         }
       },
+      {
+        "_id": "992ab4e40d46a7721b3c8282674fca4b",
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 0,
+          "cookies": [],
+          "headers": [
+            {
+              "name": "accept",
+              "value": "application/json"
+            },
+            {
+              "name": "user-agent",
+              "value": "openwhisk-client-js"
+            },
+            {
+              "name": "authorization",
+              "value": "Basic ZmFrZV9hdXRo"
+            },
+            {
+              "name": "x-namespace-id",
+              "value": "fake_ns"
+            },
+            {
+              "name": "host",
+              "value": "fake_host"
+            }
+          ],
+          "headersSize": 347,
+          "httpVersion": "HTTP/1.1",
+          "method": "GET",
+          "queryString": [],
+          "url": "https://fake_host/api/v1/namespaces/fake_ns/packages"
```

Comment:
how did you record this?

Reply:
I first recorded call to openwhisk with my credentials; and then put in all the fake stuff in the recordings

Reply:
right...but why change the host ?

Reply:
right, that was probably overkill. Would it be any trouble to leave it? The authorization is fake

Reply:
sure...just keep it...I was just curious.